Week3/homework/step2-5.js

```diff
@@ -3,10 +3,13 @@
 function multiplyAll(arr) {
   // eslint-disable-next-line
   let product = 1;
-
-  // Replace this comment and the next line with your code
-  console.log(arr, product);
-
+  let i = 0;
+  let j = 0;
+  for (i = 0; i < arr.length; i++) {
```

Comment:
You could just add the `let` here and skip the lines above. 

`for (let i = 0; i < ...)`

Reply:
Thanks! you are right!  I will update it 👍 :)